ulift-web/package-lock.json

```diff
@@ -1937,6 +1937,19 @@
         }
```

Comment:
Você vai usar npm ou yarn?  o projeto tem os arquivos do npm e do yarn, remover um dos dois

Reply:
Se for o yarn.lock ele vem por padrão pois p create-react-app utiliza o yarn

Reply:
escolhe qual vai usar